src/components/SideMenu/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { List, ListItemButton, ListItemText } from "@mui/material";
+import "./SideMenu.css";
+import React from "react";
+import {
+  AutoStoriesOutlined,
+  HomeOutlined,
+  PermIdentityOutlined,
+} from "@mui/icons-material";
+
+const SideMenu = () => {
+  const [selectedIndex, setSelectedIndex] = React.useState(0);
```

Comment:
Maybe you can import useState hook and use it directly, just to be consistent with other components in project

Reply:
Also you can check how to explicitly set what is the type of `selectedIndex`